.gitignore

```diff
@@ -14,8 +14,15 @@ out/
 config/*.yaml
 
 # binary
-oak-avs
-avs-mvp
+ap-avs
 
 .DS_Store
 node_modules
+
+# Ignores operator configuration files at root
+operator.yaml
+metadata.json
+
+# Ignores yaml configuration files in the config directory
+/config/operator.yaml
+/config/aggregator.yaml
```

Comment:
These are referred to by our docs and default names, so better just ignore all of them here.

Reply:
Okay, sticking with `config/*.yaml`